src/util.ts

```diff
@@ -0,0 +1,38 @@
+const BASE_URI = 'fly://erc1155';
+
+export function packTokenUri(namespace: string, name: string, client_id: string) {
+  const uri = new URL(BASE_URI);
+  uri.pathname = `/${namespace}/${name}/${client_id}`;
```

Comment:
I wonder if the namespace and name are really part of the base, not the path.
`fly://erc1155.name.namespace/client_id`
Totally subjective of course, and I don't feel strongly about it - just feeding back a gut reaction I had seeing the URI.

Reply:
I'd lean somewhat strongly toward keeping them part of the path rather than the host. Subdividing the host isn't well-defined AFAIK (I think the fact that DNS uses dots is outside the URI spec). I think the hierarchy of pathname fits well though.

I will say that including both the name and the ID in the path feels a bit odd (since ID is not a "child" of name). Could move one of those to the query (`?id=xxxxxx`), but I'm not sure if that ultimately makes things more or less clear.

Reply:
Thanks for thinking it through - sounds like what we have is good 👍